Lib/LoginServiceKit/LoginServiceKit.swift

```diff
@@ -65,9 +65,11 @@ public final class LoginServiceKit: NSObject {
         guard isExistLoginItems(at: path) == false else {
             return false
         }
-        guard let (list, items) = snapshot, let item = items.last else {
+        guard let (list, items) = snapshot else {
             return false
         }
+        
+        let item = unsafeBitCast(items.last ?? nil, to: LSSharedFileListItem.self)
```

Comment:
```suggestion
        let item = unsafeBitCast(items.last, to: LSSharedFileListItem.self)
```

Reply:
@dimitarnestorov Thanks for the PR 👍 It worked correctly in my environment.
Can you please incorporate Suggested Change?